README.md

```diff
@@ -3,7 +3,315 @@
 This repo contains Simplyblock CSI ([Container Storage Interface](https://github.com/container-storage-interface/))
 plugin for Kubernetes.
 
-Simplyblock CSI plugin brings high performance block storage to Kubernetes. It provisions SPDK logical volumes on storage node dynamically and enables Pods to access SPDK storage backend through NVMe-oF .
+SPDK CSI plugin brings SPDK to Kubernetes. It provisions SPDK logical volumes on storage node dynamically
```

Comment:
the changes here were removed and moved to internal folder as a part of PR: https://github.com/simplyblock-io/spdk-csi/pull/39

Please discard the changes in this file.